opentelemetry-sdk/src/testing/mod.rs

```diff
@@ -3,3 +3,6 @@ pub mod trace;
 
 #[cfg(all(feature = "testing", feature = "metrics"))]
 pub mod metrics;
+
+#[cfg(all(feature = "testing", feature = "span"))]
+pub mod span;
```

Comment:
I believe there's no need for an independent `span` feature as mentioned earlier. The in_memory_exporter can be integrated within the `trace` feature. Create a new directory named `trace` and relocate the existing `trace.rs` and `in_memory_exporter.rs` files into it. Once that's done, we can delete the `span` directory.